numpy/linalg/linalg.py

```diff
@@ -1300,19 +1300,21 @@ def svd(a, full_matrices=True, compute_uv=True):
     """
     Singular Value Decomposition.
 
-    Factors the matrix `a` as ``u * np.diag(s) * v``, where `u` and `v`
-    are unitary and `s` is a 1-d array of `a`'s singular values.
+    Factors the matrix `a` as ``u * np.diag(s) * vh``. When `a` is a 2D array,
```

Comment:
Should `*` by `@` since the latter is now meaningful?

Reply:
Good for me. Do you know if this is common or desirable in the numpy docs?